### PR TITLE
#23649 I have added the filter into the out of stock message in varia…

### PR DESCRIPTION
**Pull request by parmarhardip**
#23649 
I have added a filter please review it.

### CODE REVIEW

File: templates/single-product/add-to-cart/variable.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	<?php do_action( 'woocommerce_before_variations_form' ); ?>
 
 	<?php if ( empty( $available_variations ) && false !== $available_variations ) : ?>
-		<p class="stock out-of-stock"><?php esc_html_e( 'This product is currently out of stock and unavailable.', 'woocommerce' ); ?></p>
+        <p class="stock out-of-stock"><?php echo apply_filters( 'woocommerce_out_of_stock_message', esc_html__( 'This product is currently out of stock and unavailable.', 'woocommerce' ) ); ?></p><?php // @codingStandardsIgnoreLine ?>
```
Comment (claudiosanches) [inline templates/single-product/add-to-cart/variable.php]: `@codingStandardsIgnoreLine` is deprecated and we should avoid it, if necessary there we can ignore specific rules with `phpcs:ignore <rule>`.

**Review by claudiosanches — CHANGES_REQUESTED** [review]

File: templates/single-product/add-to-cart/variable.php (hunk 1 above)
Comment (parmarhardip) [reply templates/single-product/add-to-cart/variable.php]: Hello @claudiosanches 

Thanks, I have removed the `@codingStandardsIgnoreLine` code. 

Please review and let me know.

**Review by claudiosanches — CHANGES_REQUESTED** [review]
There's still a problem with indentation on this line, need to replace for 2 tabs.

Also you could change the code to:

```php
<p class="stock out-of-stock"><?php echo esc_html( apply_filters( 'woocommerce_out_of_stock_message', __( 'This product is currently out of stock and unavailable.', 'woocommerce' ) ) ); ?></p>
```

So will escape the output no matter what, and also fixes coding standards.

Thanks again.

**Review by claudiosanches — APPROVED** [review]
Thanks for your help!

**Pull request merged**